api/files/serializers.py

```diff
@@ -197,6 +196,15 @@ def update(self, instance, validated_data):
     def is_valid(self, **kwargs):
         return super(FileSerializer, self).is_valid(clean_html=False, **kwargs)
 
+    def get_file_guid(self, obj):
+        if obj:
+            guid = obj.get_guid()
+            if guid:
+                return guid._id
+            else:
```

Comment:
Since the else branch and the default both return the same thing, you shouldn't need the else branch. Just let it fall through.